nncf/openvino/graph/model_transformer.py

```diff
@@ -431,3 +433,34 @@ def _insert_inplace_operation(
             new_node = transformation.inplace_op_fn(output.get_node(), output.get_index())
             return (new_node.output(fn_output_port_id), fn_output_port_id)
         raise RuntimeError(f"Transform type {transform_type} is not supported")
+
+    @staticmethod
+    def _apply_bias_insertion_transformations(
+        model: ov.Model, transformations: List[OVBiasInsertionCommand]
+    ) -> ov.Model:
+        """
+        Inserts bias operation after corresponding layer.
+
+        :param transformations: List of the bias insertion transformations.
+        :returns: Transformed model with null biases.
```

Comment:
Could you please specify that inserted biases always adds zero i.e. do nothing? Or maybe it worth to feed a value of bias through `OVBiasInsertionCommand`?

Reply:
I would suggest to add this for the sake of explicitness
 
```suggestion
        """
        Inserts null bias operation after corresponding layer.

        :param transformations: List of the bias insertion transformations.
        :returns: Transformed model with null biases.
```